src/main/scala/beam/utils/csv/writers/PopulationCsvWriter.scala

```diff
@@ -11,7 +11,7 @@ import scala.collection.JavaConverters._
 object PopulationCsvWriter extends ScenarioCsvWriter {
 
   override protected val fields: Seq[String] =
-    Seq("personId", "age", "isFemale", "householdId", "houseHoldRank", "excludedModes")
+    Seq("personId", "age", "isFemale", "householdId", "householdRank", "excludedModes")
```

Comment:
Won't the renaming break the things? Or we don't have so much data created with new CSV writer?

Reply:
No. this writer has never been used. It will be used by the reader I am writing in this PR